manifests/base/deployment.yaml

```diff
@@ -113,6 +113,24 @@ spec:
                   name: jwt-secret
             - name: S3_BUCKET
               value: campaign-files
+            - name: IRIS_API_URL
+              value: https://developer.sandbox.irispay.bg/api/8
+            - name: BANK_BIC
+              value: UNCRBGSF
+            - name: PLATFORM_IBAN
+              value: test
```

Comment:
can we already add the correct IBAN? I assume that's the IBAN of the organization, which should be public anyways

Reply:
yes, BG66UNCR70001524349032